src/adapters/middleware-adapter.ts

```diff
@@ -57,8 +57,14 @@ export async function handler(event: CloudFrontRequestEvent): Promise<CloudFront
     return request;
   }
 
+  // In the case where `middleware.ts` response w/ a body, send that back to the client
```

Comment:
This may be useful to have if your middleware.ts returns data immediately to the client.